docs/docs/dev-manuals/services/frontend.mdx

```diff
@@ -0,0 +1,161 @@
+# Frontend
+
+:::info
+This document is still under development.
+:::
+
+**This documentation is primarily intended for software developers.**
+
+First things first, the GitHub **repository** can be found [here](https://github.com/MiSArch/frontend), the Gropius component for **issue tracking** can be found [here](https://frontend.gropius.duckdns.org/components/9d12d6ac-e85a-407c-bdff-a8321fb75d3a).
+
+## Technology Stack
+
+- JavaScript Framework: [Vue](https://vuejs.org/guide/introduction.html#what-is-vue)
+- Programming Languages: mainly TypeScript and standard HTML within Vue components
+- UI Library: [Vuetify](https://vuetifyjs.com/en/)
+- Package Manager: [Yarn](https://yarnpkg.com) -- Instructions for the most important yarn commands can be found in the README.md in the repository.
+- Tooling: ESLint, Prettier, GraphQL Codegen
+
+## Repository Structure
+
+<div className="repository-structure">
+- `/src/`: The source code
+  - `assets/`: Idk, tbh, this was generated somehow.
+  - `components/`: [Single-File Components](https://vuejs.org/glossary/#single-file-component) that can be reused -- this is basically an internal UI components library.
+  - `graphql/`: Source code files that form the GraphQL client for the calls to the gateway.
+  - `layouts/`: [SFCs](https://vuejs.org/glossary/#sfc) responsible for the general layout of a page of the web application.
+  - `plugins/`: Plugins related source code.
+  - `router/`: The app-specific logic of the router is implemented here, enabling navigation between the many views.
```

Comment:
```suggestion
  - `router/`: navigation between the available views, so what is displayed where.
```

Reply:
Hm, the thing is that it's not only navigation but **how** exactly the navigation is supposed to work. The Vue Router works out of the box and with a minimal setup you can set up basic navigation. However, one can really extend the Router's logic when it comes to access rights to certain routes, for instance.

How about: `Logic and rules for the Vue Router. This enables navigation between the views including access rights checking and redirects.`

Reply:
Hm, the thing is that it's not only navigation but **how** exactly the navigation is supposed to work. The Vue Router works out of the box and with a minimal setup you can set up basic navigation. However, one can really extend the Router's logic when it comes to access rights to certain routes, for instance.
How about: Logic and rules for the Vue Router. This enables navigation between the views including access rights checking and redirects.